MD/md-n/README.md

```diff
@@ -0,0 +1,219 @@
+
+| Feature                        | Fully Synchronous            | Partially Synchronous               |
+|-------------------------------|-----------------------------------|----------------------------------------|
+| **Timing Assumption**         | Fixed known bounds                | Unknown bounds before GST              |
+| **Liveness Guarantee**        | Always (if bounds hold)           | Only eventually (after GST)            |
+| **Safety Guarantee**          | Depends on strict timing, traditional Byzantine assumption violated          | Holds even under asynchrony            |
+| **Fault Tolerance**           | Assumptions must be modified to allow a non-supermajority fork to exist, hence not traditionally BFT  | Designed to tolerate Byzantine faults  |
+| **Performance**               | High under tight control          | Adaptive but may have delays           | 
+
+### A2: PBFT and Perfectness
+
+[PBFT](https://pmg.csail.mit.edu/papers/osdi99.pdf) introduced a protocol which ensures consecutive rounds of consensus intersect in at least one unit of honest voting power via the Generalized Pigeon Hole Principle. The transitivity of this property ensures that a chain of consensus rounds must intersect in at least one unit of honest voting power. 
+
+Under an [experts model](https://people.csail.mit.edu/ghaffari/AA19/AAscript.pdf?utm_source=chatgpt.com) understanding of PBFT systems, we can make an assumption that there is a perfect expert and thus that this perfect expert is identified by consensus. This, in turn, renders PBFT a lossless approximate algorithm.
+
+If we consider allowing forks, however, it is initially unclear how perfectness translates. Naively, our assumption that there is globally one perfect expert means that there is one **perfect** fork. 
+
+We can, however, apply a reduced criterion and state that a each **fork** be comprised of a chain of consensus rounds each of which intersect in one unit of honest voting power w.r.t. to the fork itself. We call this **fork-perfectness**.
+
+Preserving **fork-perfectness** reduces to ensuring the fraction of stake which decided the original fork maintains a supermajority. If this does not occur, then it is impossible to have the guaranteed intersection. 
+
+### A3: Awareness of Minority Decisions
+
+The requirement of **full-synchronicity** means that if a supermajority decision is not made by Global Stabilization Time, some form of minority decision must effectively be made. This does not inherently mean that a Byzantine fraction of voting power may decide the global state. For example, if long-lived forks are preserved, there are multiple global states and eventual supermajority consistency can be achieved via a reconstitution of supermajority stake on an honest fork. In other words, while a **partially-synchronous** system renders eventual liveness and guaranteed safety, a **full-synchronous** system can render guaranteed liveness and eventual safety. 
```

Comment:
> For example, if long-lived forks are preserved, there are multiple global states and eventual supermajority consistency can be achieved via a reconstitution of supermajority stake on an honest fork.

You are just describing a system that favors liveness over safety. in the system you describe there is never any finality. Which makes any quasi-synchronous decission void. you may as well not take a decission at point Delta.